flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/internals/KinesisDataFetcher.java

```diff
@@ -542,6 +545,16 @@ public int registerNewSubscribedShardState(KinesisStreamShardState newSubscribed
 		}
 	}
 
+	/**
+	 * Registers a metric group associated with the shard id of the provided {@link KinesisStreamShardState shardState}.
+	 */
+	private MetricGroup buildMetricGroupForShard(KinesisStreamShardState shardState) {
+		return runtimeContext
+			.getMetricGroup()
+			.addGroup("Kinesis")
```

Comment:
is this necessary?

Reply:
I think so. Just following the convention of other components (e.g. [kafka consumer](https://github.com/apache/flink/blob/00ad0eb120026817c79a5fed2c71dd6aa10e9ba6/flink-connectors/flink-connector-kafka-0.9/src/main/java/org/apache/flink/streaming/connectors/kafka/internal/Kafka09Fetcher.java#L101))

Reply:
Ah I see, that makes sense then.

Could you then name it as `KinesisConsumer`, to be more consistent?